_rules/css-restrict-orientation-b33eff.md

```diff
@@ -23,7 +23,10 @@ acknowledgments:
 
 ## Applicability
 
-This rule applies to any [HTML element][] that is [visible](#visible) and has a CSS [transform](https://www.w3.org/TR/css-transforms/#propdef-transform) property that is applied conditionally on the [orientation](https://www.w3.org/TR/css3-mediaqueries/#orientation) [media feature](https://www.w3.org/TR/css3-mediaqueries/#media1) with a value of `landscape` or `portrait`, where the CSS [transform](https://www.w3.org/TR/css-transforms/#propdef-transform) property has any of the below [transformation functions](https://www.w3.org/TR/css-transforms/#transform-functions):
+This rule applies to any [HTML element][] that is [visible](#visible) and has one of the following CSS properties applied conditionally on the [orientation](https://www.w3.org/TR/css3-mediaqueries/#orientation) [media feature](https://www.w3.org/TR/css3-mediaqueries/#media1) with a value of `landscape` or `portrait`:
+
+1. the CSS [rotate](https://www.w3.org/TR/css-transforms-2/#individual-transforms) property
```

Comment:
Actually, we should also change the title of the rule. "Orientation of the page is not restricted using CSS **transform** property" is not correct any more since it's "transform or rotate".

Reply:
I've updated the name of the rule to 'Orientation of the page is not restricted using CSS transforms'

That's technically accurate since the rule references to the CSS Transforms spec, and means other transforms can be added without changing the name of the rule again

Is everyone ok with that?